index.html

```diff
@@ -0,0 +1,233 @@
+         <li class="lists">html</li>
+         <li class="lists">css</li>
+         <li class="lists">javascript</li>
+       </ul> 
+       <button id="bttn">see project</button>
+    </section>
+    <section class="card_item">
+      <div class="snap">
+        <img src="images/snap2/Snapshoot Portfolio.png" alt="">
+      </div>
+      <h1 class="tex">Multi-Post Stories</h1>
+      <ul class="list">
+        <li class="canopy"><b> CANOPY</b></li>
+        <li> <img src="images/cano/Counter.png" alt=""> </li>
+        <li class="bac">Back End Dev</li>
+        <li><img src="images/cano/Counter.png" alt=""></li>
+        <li class="bac">2015</li>
+      </ul>
+      <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="text">
+          <li class="lists">html</li>
+          <li class="lists">css</li>
+          <li class="lists">javascript</li>
+        </ul>
+        <button id="bttn">see project</button>
+    </section>
+    <section class="card_item">
+      <div class="snap">
+        <img src="images/snap3/Snapshoot Portfolio.png" alt="">
+      </div>
+      <h1 class="tex">Tonic</h1>
+      <ul class="list">
+        <li class="canopy"><b> CANOPY</b></li>
+        <li><img src="images/cano/Counter.png" alt=""></li>
+        <li class="bac">Back End Dev</li>
+        <li><img src="images/cano/Counter.png" alt=""></li>
+        <li class="bac">2015</li>
+      </ul>
+      <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="text">
+        <li class="lists">html</li>
+        <li class="lists">css</li>
+        <li class="lists">javascript</li>
+      </ul> 
+      <button id="bttn">see project</button>
+    </section>
+    <section class="card_item">
+      <div class="snap">
+        <img src="images/snap4/Snapshoot Portfolio.png" alt="">
+      </div>
+      <h1 class="tex"> Multi-Post Stories</h1>
+      <ul class="list">
+        <li class="canopy"><b> CANOPY</b></li>
+        <li><img src="images/cano/Counter.png" alt=""></li>
+        <li class="bac">Back End Dev</li>
+        <li><img src="images/cano/Counter.png" alt=""></li>
+        <li class="bac">2015</li>
+      </ul>
+      <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="text">
+        <li class="lists">html</li>
+        <li class="lists">css</li>
+        <li class="lists">javascript</li>
+      </ul>
+      <button id="bttn">see project</button>
+    </section>
+  </section>
+
+  <section class="section6">
+    <div>
+      <h2 class="tex">About <br> Myself</h2><br>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p><br>
+      <p class="lets"> LET'S CONNECT
```

Comment:
- [ ]  Kindly I recommend you to use `hx` header tag instead of `p` tag for "Let's connect" to make your HTML code semantic correct.